src/foam/parse/parse.js

```diff
@@ -484,6 +505,38 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.parse',
+  name: 'Chars1',
+
+  documentation: `Matches against any of the chars specified
+    in the argument string and returns the matched char.`,
+
+  properties: [
+    {
+      name: 'string',
+      final: true
+    }
+  ],
+
+  methods: [
+    function parse(ps) {
+      if (!ps.valid) return undefined;
+      var ind = this.string.indexOf(ps.head);
+      return ind !== -1 ? ps.tail.setValue(this.string.charAt(ind)) : undefined;
```

Comment:
I think this will be functionally identical to the exisitng Chars.  The implied `value` of any pstream is the `ps.head` of the pstream that came before.

if ps.head === 'c' then ps.tail.value === 'c'.

Reply:
Thanks! Somehow I missed that